pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -456,19 +469,28 @@ func ConnectClickHouse(config *ClickHouseConfig) (*sql.DB, error) {
 	return connect, nil
 }
 
-func parseDatabaseURL(dbUrl string) (protocol, string, error) {
+func parseDatabaseURL(dbUrl string, secure bool) (clickhouse.Protocol, string, error) {
 	u, err := url.Parse(dbUrl)
 	if err != nil {
-		return protocolUnknown, "", fmt.Errorf("failed to parse ClickHouse database URL: %w", err)
+		return -1, "", fmt.Errorf("failed to parse ClickHouse database URL: %w", err)
```

Comment:
Maybe define const instead of directly using -1 here?

Reply:
I missed one. Updated. Thanks!